hieradata/users.yaml

```diff
@@ -23,7 +23,8 @@ gds_accounts::accounts:
         ssh_key: AAAAB3NzaC1yc2EAAAADAQABAAACAQDozuKT8j3aciI4z86/Vf74zsZviHQpCqLJXMCG6Cl1Pf8dOZYnDbI4hPGnpBRNrticmTPzGeXG9bsMe30x3+NMupkIZhfLQKJKmWnoLoEse8kUx8q5/eH4oHybuXYisBT2n3ORq4JUQbjDmjTJfOELCL0os8554RSMlIjzhS9U11z1NwyjCOeDSA0GzwIhEg0phc9zNpMiD8koj89alCpFiktm4EdjXmhGyQVNSffRxwwIgAc/XswL2+2AfZ6vR2xJQMMNM2y2XkIKsaCsRYOVN0UjOTuvEgg5CWrniOy+3nipcSk7YzjUv7ozGIdIaUTSVwzRtD3B9zjobQHUIOOum1fDnKaJJIQba0SwFYe2mLmPK0DN5SyfE884jsOk+Tj8KpKFcVof071GFC9e+2TdPGkFqkVUZFXhjDdV8LbJENtVKhDnv6C8Foo6Ir9RG5gzFaGRrzeKeQkgubH7DJyFAHUaeVVrzK+24ca0ef9xZvFLa0KwWR2Cv6xvS3kUuv6JSvInCW6sdfmMx18M3kqPJcn5TQljEJg5Vz7/lDORUP3/pmgv5sKYrDIWRTIbfMGCrh1S91fPWLGnOIpOjRVF9Dseb4gkax3dZ2n5lQZp0xIUXXHVT7eNKcWpCAqmxvPy4cnZCKD9WzgjT2ZP0ZhaXTob6Uy57yewTVzTaqvTlw==
     isabelllong:
         comment: Isabell Long
-        ssh_key: AAAAB3NzaC1yc2EAAAADAQABAAACAQDUbyr7etlWC/BUYSM3Y447UHGVYULrdu8p6/kbyqPV9KF1Ncn83WuhC/sghWIKOW5famspMcLJS2Sorc+k9FyX3qmgMG/ZNRHNN7xbMfdtJSgk4U74bLsTVg00OrTOTRpt984R7m3HQ5wHWIDazZximbl4AHy9vVwLKJ8HT7jwYGja6WFdkJiqscRgoFNE/IEjqntFXZmnILw2TGD4eCypRI0Hf0R5MWhAQkp98p8ul/935+TqC1KQEZkAY89d5imhlXjSr23JFfF+XNtyHkUG+E7KN5O0tUPyVRPfGXk3WYKsMtDqkhoIkT5wokSt9gOGkTbOmhxuHQQZjZh3n07eIBKVR8GS9Vn7g+Sr125zXzJ08mWc/jekoTxMhyDlPPO7kxkU5ucALw5Xt1oST8R/0XRPJgH0cMzkoFFJZt+hUbovMPYlRiR91I76uGb898Kn5Vp0vefVccyaMDIjcoBtgIK6RPinWrzSNAAGBhXORQ2UpuQUwH+mfkjsoWI+XO8ShRKxgeOTkTTTD8+2br1lvWaurSyhybIbua8L8GazHO5GXxtqdGWDTI9C1hfPr1P8vXTecJ/Dmv/xUdFNfoiHrCSN+IG+sTr0W1rq6RvkLzFE8OkqFfF7dnqwesBOi5glkjZsRBrFfYxKFg0ebtwlAuc/u79LNpYgf6F3WpTwBw==
+        ssh_key: AAAAB3NzaC1yc2EAAAADAQABAAACAQC7HfCT0IusPqKxxEbap5yK1xBrKUpSB8B8wdiK5LgSWbaf78oY25nK5LE6i+NTFVt1Kxpd5uYs7/X3BquvdDrnkRVeJyfqDpJFSSSQBHiHSoVLv+EYzEYVRac7BT/riWSsNrGRGhAMMo84rWmSFmQxDBgJ4XMhv4Z0/2eBNRK+uZv+PHiOrWu8uAGmyr5s8bj93UqzafaoUaCcUBoWMGGk0pkmThlRAThpduvOvYqlX4YIbJ3vgEUfxCWhedmgbo2n4UXq8YISE46b5m9YmH31ZDifZ+WF3QXHsGQZmsB5TgSjTLHQGqoI7XpfTd1T1dGnIFT8ykBVrOFa7N5/AVn3AKBT3F63Ct9rMqdx9oIAn7d8OJQrFQfA1j+8fnUysEsO7lNALA/M0rAwAPwwC+YGwaspwTEzjFFK5/jPFhk+Z8n9PHZflxPN6bhRwv7PhK6tCmJ3jEogBSljE+/aLMS32HaaBjtxQO4AcjdxAB0ICsoY+Jl3kldJPudttrG6APufr0pSemNRPz6sVoxsph85CxdImKt+EDLpOxq25D+gQv2ACHRuDsqrZO2xDt357nqEoJNEL9W6BBDyeC2AIWA+59v4z7NveZf58Y8YhBZ5ZJdHOUP545V9tDBQDuttWYR/JoU4iPegKPuYZuayPYyYIcqSF2HIbbQ31qvyLSTs2w==
+
```

Comment:
There's an extra line in here which I think might cause the YAML hash to be upset.